gillespy2/core/model.py

```diff
@@ -229,6 +239,43 @@ def decorate(header):
 
         return print_string
 
+    def add(self, components):
+        """
+        Reactions, Events, Rate Rules, and Assignment Rules must come after Species and Parameters.
```

Comment:
Expand the docstring here with info on valid `components` types and a rundown on what the function does.